src/components/MenuPanel/OrganizationSwitcher/Favorites.js

```diff
@@ -125,10 +119,14 @@ class Favorites extends React.Component {
       }
     })
 
-    const currentItem = allItems.find(org => org.address === currentDao.address)
-    const otherItems = allItems.filter(
-      org => org.address !== currentDao.address
-    )
+    const favoriteItems = [...allItems].sort((org, org2) => {
+      if (addressesEqual(org.address === currentDao.address)) {
+        return -1
+      } else if (addressesEqual(org.address === currentDao.address)) {
```

Comment:
`org` => `org2` (not sure if it should be for `1` or `-1`), and we don’t need the `else`:

```suggestion
      }
      if (addressesEqual(org2.address === currentDao.address)) {
```

Reply:
Ah yes, good catch! It should be `org2` :). I also completely messed up the arguments to `addressesEqual`; I'm not sure how this even worked when testing :/

Reply:
Oh yes that too :laughing:

Reply:
Fixed in cb54597, and also applied sorting based on the name.